pkgs/development/tools/rust/cargo-show-asm/default.nix

```diff
@@ -21,14 +24,25 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-S7OpHNjiTfQg7aPmHEx6Q/OV5QA9pB29F3MTIeiLAXg=";
 
-  nativeBuildInputs = [ pkg-config installShellFiles ];
-  buildInputs = [ openssl ];
+  nativeBuildInputs = [
+    curl.dev
```

Comment:
Is both `curl.dev` and `curl` below required? Using it in both `nativeBuildInputs` and `buildInputs` is kind of weird.

Reply:
curl-sys calls curl-config to check for features and falls back to vendorred version when it fails